### PR TITLE
DFBUGS-867: [RDR] UI blocks a DR operation on an appset based workload

### PR DESCRIPTION
**Pull request by GowthamShanmugam**

### CODE REVIEW

File: packages/mco/components/modals/app-failover-relocate/helper/error-messages.tsx [hunk 1]
```diff
@@ -287,8 +287,8 @@ export const evaluateErrorMessage = (
   errorMessage: ErrorMessageType,
   includeWarning: boolean = false
 ) => {
-  if (!includeWarning ? errorMessage < 20 : true) {
-    return errorMessage;
+  if (!errorMessage || (!includeWarning && errorMessage >= 20)) {
```
Comment (SanjalKatiyar) [inline packages/mco/components/modals/app-failover-relocate/helper/error-messages.tsx:290]: say `errorMessage` is `22` (delay warning), wonl't https://github.com/red-hat-storage/odf-console/blob/master/packages/mco/components/modals/app-failover-relocate/failover-relocate-modal-body.tsx#L327 `evaluateErrorMessage(errorMessage, true) > -1` be false and we don't display any warning message on the modal ??

[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply packages/mco/components/modals/app-failover-relocate/helper/error-messages.tsx:290]: i am starting enum number from 1:
export enum ErrorMessageType {
  // Priority wise error messages
  DR_IS_NOT_ENABLED_FAILOVER = 1,

[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply packages/mco/components/modals/app-failover-relocate/helper/error-messages.tsx:290]: good catch, i will change enum from 1

[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply packages/mco/components/modals/app-failover-relocate/helper/error-messages.tsx:290]: done

**Pull request merged**